.changeset/lazy-radios-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+"comet-api-v9": major
```

Comment:
why is this a major change? shouldnt this be a minor?

Reply:
Version was changed.

3075824a6c3b74eb1cf49f8668c92d36fa49ad32

Reply:
Changed doc for changeset

86863103cafd2f509635c35a6d4d1236956ad438